typescript/infra/scripts/safes/create-safe.ts

```diff
@@ -54,10 +54,18 @@ async function main() {
   const safeAddress = await safe.getAddress();
 
   console.log(`Safe address: ${safeAddress}`);
-  console.log(
-    `Safe url: ${safeTxServiceUrl}/home?safe=${chain}:${safeAddress}`,
-  );
+  console.log(`Safe url: ${safeHomeUrl}/home?safe=${chain}:${safeAddress}`);
   console.log('url may not be correct, please check by following the link');
+
+  try {
+    const response = await fetch(`${safeHomeUrl}/home`);
+    const resultsJson = await response.json();
```

Comment:
does this work?
`https://safe.mainnet.frax.com/home` doesn't give me JSON

Annoyingly I feel like it changes with each deployment, e.g. for frax the endpoint is `https://safe.mainnet.frax.com/cgw/v1/chains`, for the official gnosis safe it's `https://safe-client.safe.global/v1/chains` :(

Reply:
Hmm yes it can vary per deployment